migrations/versions/3417aec47fe2_encrypt_refresh_tokens.py

```diff
@@ -0,0 +1,52 @@
+"""Encrypt refresh tokens
```

Comment:
The changes from #40 are appearing in this PR diff, but are absent when [diffing the branches](https://github.com/uc-cdis/workspace-token-service/compare/master...add-aggregate-authz-endpoint).

```
13:38:08 johnmccann Johns-MBP  /Users/johnmccann/Programs/workspace-token-service master
$ git pull
Already up to date.
13:38:10 johnmccann Johns-MBP  /Users/johnmccann/Programs/workspace-token-service master
$ git diff master...add-aggregate-authz-endpoint migrations/
13:39:23 johnmccann Johns-MBP  /Users/johnmccann/Programs/workspace-token-service master
$
```